.eslintrc

```diff
@@ -24,6 +24,9 @@
             "allowTemplateLiterals": false
         }],
         "brace-style": 2,
+        "keyword-spacing": ["error", {
+            "after": true
```

Comment:
After is already `true` [by default](https://eslint.org/docs/latest/rules/keyword-spacing#options), so this line has no effect.